.github/workflows/sync-prod-from-stage.yaml

```diff
@@ -10,8 +10,10 @@ jobs:
         uses: actions/checkout@v4
       - name: Opening pull request
         id: pull
-        uses: tretuna/sync-branches@1.4.0
+        # We need main for PULL_REQUEST_AUTO_MERGE_METHOD, which is currently not yet released.
+        uses: tretuna/sync-branches@main
```

Comment:
@stehessel Should we pin it to commit to avoid unexpected changes in main to break our actions?

Reply:
See the comment, we need `main` until they release the `PULL_REQUEST_AUTO_MERGE_METHOD` feature.